src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://kit.fontawesome.com/d713187366.js" crossorigin="anonymous"></script>
+  <title>To Do List</title>
+</head>
+
+<body>
+  <main>
+    <h1>To Do List</h1>
+    <ul>
+      <li><span>Today's To Do</span><i class="fa-solid fa-arrows-rotate"></i></li>
+      <li id="form-item">
+        <form name="addTaskForm" id="addTaskForm" autocomplete="off">
+          <input type="text" name="description" required id="description" placeholder="Add to your list...">
+          <button type="submit"><i class="fa-solid fa-thumbtack"></i></button>
+        </form>
+      </li>
+      <li class="list-footer">
+        Clear all completed
```

Comment:
you used a good structure of HTML semantics here, it is a good idea to use ul and li for this project. 🎖️ 
I see a small change is required for a better result for HTML semantics rule 🤞 . I prefer to use a button inside this li and put "clear all completed" text in it. You can apply same event 👌 .

Reply:
> Correction

Change the `li | clearAllCompleted` tag to a `button`